### PR TITLE
testing/openresty: new aport

### PR DESCRIPTION
**Pull request by mengskysama**
https://openresty.org/
A Fast and Scalable Web Platform by Extending NGINX with LuaJIT

### CODE REVIEW

File: testing/openresty/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+arch="all"
+license="BSD"
+depends="gd geoip libxslt zlib perl !nginx"
+makedepends="gd-dev geoip-dev libxslt-dev linux-headers perl-dev readline-dev zlib-dev libgcc"
+install=""
+subpackages=""
+
+source="https://openresty.org/download/$pkgname-$pkgver.tar.gz
+openssl-$resty_openssl_ver.tar.gz::https://www.openssl.org/source/openssl-$resty_openssl_ver.tar.gz
+pcre-$resty_pcre_ver.tar.gz::https://ftp.pcre.org/pub/pcre/pcre-$resty_pcre_ver.tar.gz"
```
Comment (jirutka) [inline testing/openresty/APKBUILD]: This is not acceptable, please use system-provided libressl (or openssl if needed) and pcre.

File: testing/openresty/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# Contributor: Yichun Zhang <agentzh@gmail.com>
+# Maintainer: Yichun Zhang <agentzh@gmail.com>
+pkgname=openresty
+pkgver=1.13.6.1
+pkgrel=0
+pkgdesc="Scalable Web Platform by Extending NGINX with Lua"
+url="https://openresty.org/"
+arch="all"
+license="BSD"
```
Comment (Ikke) [inline testing/openresty/APKBUILD]: The license should be in the format provided by SPDX. In this case, is 'BSD-3-Clause`.

File: testing/openresty/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+depends="gd geoip libxslt zlib openssl pcre !nginx"
+makedepends="gd-dev openssl-dev pcre-dev geoip-dev libxslt-dev linux-headers readline-dev zlib-dev libgcc"
+install=""
+subpackages=""
+
+source="https://openresty.org/download/$pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+prepare() {
```
Comment (Ikke) [inline testing/openresty/APKBUILD]: This is redundant and should be removed.

File: testing/openresty/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+pkgname=openresty
+pkgver=1.13.6.1
+pkgrel=0
+pkgdesc="Scalable Web Platform by Extending NGINX with Lua"
+url="https://openresty.org/"
+arch="all"
+license="BSD"
+depends="gd geoip libxslt zlib openssl pcre !nginx"
+makedepends="gd-dev openssl-dev pcre-dev geoip-dev libxslt-dev linux-headers readline-dev zlib-dev libgcc"
+install=""
```
Comment (Ikke) [inline testing/openresty/APKBUILD]: Empty variables can be removed

**Comment by Ikke** [comment]
Closing this, as [openresty][] has been submitted in testing already.

[openresty]:https://pkgs.alpinelinux.org/package/edge/testing/x86_64/openresty

**Pull request closed without merging**